core/src/Kokkos_View.hpp

```diff
@@ -814,14 +814,14 @@ class View : public ViewTraits<DataType, Properties...> {
 
   template <typename... Is>
   static KOKKOS_FUNCTION void check_access_member_function_valid_args(Is...) {
-    static_assert(traits::rank <= sizeof...(Is), "");
+    static_assert(rank <= sizeof...(Is), "");
```

Comment:
`traits::rank[_dynamic_rank]` are still valid but it is cleaner without qualifying.